packages/slidev/node/cli.ts

```diff
@@ -53,8 +53,7 @@ cli.command(
       describe: 'open in browser',
     })
     .option('remote', {
-      default: false,
-      type: 'boolean',
+      type: 'string',
```

Comment:
Can this be an optional string?

Reply:
Yes.
The only option I found for that is to use `'string'` without any default value:
* if no option => `remote=undefined` => remote not enabled
* if `--remote` => `remote=''` (empty string) => remote enabled and public
* if `--remote password` => `remote='password'` => remote enabled with password